mne_bids/path.py

```diff
@@ -626,6 +626,78 @@ def mkdir(self, exist_ok=True):
         self.directory.mkdir(parents=True, exist_ok=exist_ok)
         return self
 
+    def rm(self, safe_remove=True, verbose=True):
+        """Safely delete a set of files inside BIDS dataset.
+
+        Deleting a scan that conforms to the bids-validator, will
+        delete both the row in the ``*scans.tsv`` file, and also
+        the corresponding sidecar files and the data file itself.
+
+        Deleting all files of a subject will update the
+        ``*participants.tsv`` file.
+        """
+        # only proceed if root is defined
+        if self.root is None:
+            raise RuntimeError('Cannot remove files from BIDS dataset')
+
+        # get file paths to delete
+        paths_to_delete = self.match()
+        pretty_paths = '\n'.join(paths_to_delete)
+
+        if verbose:
+            print(f'Deleting the following files:\n '
+                  f'{pretty_paths}')
```

Comment:
Please use `logger.info()` and remove the `if`

Reply:
yes